qemu/tests/pci_hotplug.py

```diff
@@ -53,7 +53,7 @@ def pci_add(pci_add_cmd):
 
     def is_supported_device(dev):
         # Probe qemu to verify what is the supported syntax for PCI hotplug
-        cmd_output = vm.monitor.human_monitor_cmd("?")
+        cmd_output = vm.monitor.human_monitor_cmd("?", debug=False)
```

Comment:
qmp monitor is offially supported. 
This line is used to find the it has the device_add commands in hmp,then it will continue the test ,it will not know whether the commands exists in qmp monitor ,then we might miss bugs